src/index.html

```diff
@@ -214,52 +214,74 @@
                         </p>
-                        <p>To submit data to EVA please download this  <a href="files/SubmissionTemplate.xlsx" target="_blank"  onClick="ga('send', 'event', { eventCategory: 'Download', eventAction: 'Click', eventLabel: 'SubmissionTemplate.xlsx'});">template</a>, fill in and send by email to <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a>.</p>
                     </div>
 
                     <h4><span class="icon icon-functional" data-icon="_"></span> Submit</h4>
                     <div>
-                        <p>Upload your data files to your private submission upload account or directly to the <a href="mailto:eva-helpdesk@ebi.ac.uk?subject=Sequence submissionto EVA&amp;body=**We will provide you with a submission package metadata templates guides for completing templates**" target="_top">eva-helpdesk@ebi.ac.uk</a>.<br/>
-                            To improve submission processing time, users are encouraged to prevalidate submitted files using the <a href="https://github.com/EBIvariation/vcf-validator" target="_blank">EVA VCF validation suite</a>; more  information can be found at our <a href="?FAQ#collapseThree">FAQs</a>.</p>
+                        <p>Upload your VCF file(s), associated data file(s) and EVA metadata template to your private EVA FTP location.</p>
                     </div>
 
-                    <h3>Converting genetic variation data to VCF</h3>
+                    <h4><span class="icon icon-functional" data-icon="k" aria-hidden="true"></span> Receive</h4>
+                    <div>
+                      <p>The EVA aims to process submissions within two business days. Accession numbers shall be sent via email to the submitter upon successful archival of the deposited data.</p>
+                    </div>
+
+                    <h3>Clinically Relevant Genetic Variation Data</h3>
                     <div>
                         <p>
-                            We advise submitters to consult the <a href="http://samtools.github.io/hts-specs/VCFv4.3.pdf">VCF specification guidelines</a> when converting data to VCF in order to ensure that file(s) generated are valid, as this is a requirement of EVA submission.
+                            The EVA strongly recommends submission of clinically relevant genetic variant data (i.e. data that relates genetic variation(s) with clinical significance (e.g. pathogenic, benign, etc.) values) to the ClinVar resource at <a href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank">NCBI</a>.
                         </p>
+                        <p>Submitters unsure of the most relevant resource to archive their genetic variation data are encouraged to first contact <a href="mailto:eva-helpdesk@ebi.ac.uk?subject=Clinical submission to EVA" target="_top">eva-helpdesk@ebi.ac.uk</a> to discuss their situation.</p>
+                    </div>
+
+                    <h3>Converting genetic variation data to VCF</h3>

```

Comment:
I actually thought that "Clinically Relevant Genetic Variation Data" and "Converting genetic variation data to VCF" were placed below "Data requirements" (not sure we need to say "EVA data requirements"). Definitely the conversion to VCF should be there, to avoid scaring people who still use PLINK files or any other format.

I would suggest the following header structure:

* Data requirements
   * Converting generic variation data to VCF
   * Clinically relevant genetic variation data (remove capitals, the rest of headers don't use them)
* Key stages of EVA submissions